packages/server/lib/gui/events.coffee

```diff
@@ -199,13 +200,17 @@ handleEvent = (options, bus, event, id, type, arg) ->
       onWarning = (warning) ->
         bus.emit("project:warning", errors.clone(warning, {html: true}))
 
-      openProject.create(arg, options, {
-        onFocusTests: onFocusTests
-        onSpecChanged: onSpecChanged
-        onSettingsChanged: onSettingsChanged
-        onError: onError
-        onWarning: onWarning
-      })
+      (if options.browser then browsers.ensureAndGetByNameOrPath(options.browser, true) else browsers.get())
```

Comment:
this is a long 1-liner and is best to call another function to return one or the other

Reply:
maybe like `findBrowserObjects` or `getOneOrAllBrowsers`